app/controllers/subscriptions_controller.rb

```diff
@@ -15,6 +15,7 @@ def new
 
     if params[:email]
       @email = params[:email]
+      @subscription = Subscription.find_by(email: @email)
```

Comment:
With just someone's email address I can now see if they have a trial and how many days left on that trial. It's not exactly a huge information leak but we still shouldn't do it.